### PR TITLE
Added swift 3 support

### PR DESCRIPTION
**Pull request by mruvim**
Converted to swift 3.
Changed output path of output.mp4 to /tmp instead of /Documents, for reasons please check [iOS Data Storage Guidelines](https://developer.apple.com/icloud/documentation/data-storage/index.html)

### CODE REVIEW

**Comment by blackknight101** [comment]
Hey mruvim: why change path of uotput.mp4 to /tmp instead of /Documents
I see the rule
Data that is used only temporarily should be stored in the <Application_Home>/tmp directory. Although these files are not backed up to iCloud, remember to delete those files when you are done with them so that they do not continue to consume space on the user’s device.
I think this path exactly for this rule ?

**Comment by mruvim** [comment]
@hoangtam101 Documents directory is automatically backed up to iCloud on devices which have iCloud enabled. Output video can always be regenerated from original file therefore there is no need to store it in docs directory. Storing in /tmp directory will allow system to clean up files in case system needs more memory. We need to make sure we're using users memory responsibly.

**Comment by blackknight101** [comment]
@mruvim Thank you. i got it.

**Comment by blackknight101** [comment]
@mruvim Hi Mruvim: I just compare my code and your code was submitted. But this code not work on iPhone 7.
Are you check it on iPhone 7 with XCode 8 ?
Thank you.

**Comment by mruvim** [comment]
@hoangtam101 you are right, for some reason when compiled in Xcode 8 on iOS 10 sim video doesn't work, but it does work on actual device. When i'll have time i'll investigate what's going on with iOS 10 sim

**Comment by blackknight101** [comment]
@mruvim Thank you for your reply.
I'm just run on device ios 10. this code work well on device.
Thank you

**Review by vibrito — APPROVED** [review]

File: VideoSplashKit/Source/VideoSplashViewController.swift [hunk 1]
```diff
@@ -11,160 +11,115 @@ import MediaPlayer
...
 public enum ScalingMode {
-  case Resize
-  case ResizeAspect
-  case ResizeAspectFill
+    case resize
+    case resizeAspect
+    case resizeAspectFill
 }
 
 public class VideoSplashViewController: UIViewController {
```
Comment (kumabook) [inline VideoSplashKit/Source/VideoSplashViewController.swift:19]: I  think the access modifier should be `open` instead of `public`.

Comment (AlexanderBollbach) [reply VideoSplashKit/Source/VideoSplashViewController.swift:19]: yes, i've pod'd this into my project and cannot subclass in xcode8/swift3

**Review by stevekim0417 — APPROVED** [review]

**Review by hemangshah — APPROVED** [review]

**Pull request merged**